plugins/gopass.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: gopass
+spec:
+  version: "v0.0.3"
+  platforms:
+    - selector:
+        matchExpressions:
+          - {key: os, operator: In, values: [darwin, linux]}
+      uri: https://github.com/gopasspw/kubectl-gopass/archive/v0.0.3.zip
+      sha256: "ae4d2aa8ec061aa6916b2120fa3caf48d1cee328bab7cd263a03f8b1aeec22f8"
+      files:
+        - from: "/kubectl-gopass-*/kubectl-gopass"
+          to: "."
+      bin: "./kubectl-gopass"
+  shortDescription: Imports secrets from gopass
+  homepage: https://github.com/gopasspw/kubectl-gopass
+  caveats: |
+    This plugin requires bash and gopass to be set up correctly
+  description: |
```

Comment:
Most plugins also give some basic usage instructions and how to get help.

Reply:
I think it's ok if --help/-h works.

Reply:
I can copy / paste the usage output here, but not sure what you will gain. I think the usage is quite obvious for gopass users right?